charms/k8s/charmcraft.yaml

```diff
@@ -21,3 +59,21 @@ config:
 parts:
   charm:
     build-packages: [git]
+    charm-entrypoint: worker/src/charm.py
+  lib:
+    # move the ./worker/lib path to ./lib since 
+    # charmcraft assumes it to be there once the charm runs
+    after: [charm]
+    plugin: nil
+    source: ./
+    override-prime: |
+      rm -rf $CRAFT_PRIME/lib
+      mv $CRAFT_PRIME/worker/lib $CRAFT_PRIME/lib
```

Comment:
So, it seems that charmcraft expects the lib folder to always been at the same path level as charmcraft.yaml and other "charm files" when run on the unit. 

This moves the lib/ folder to the top level during the charm build process.